README.md

```diff
@@ -1,9 +1,9 @@
 # FINALE
 
-FINALE is a lightweight web app to view GCode. FINALE stands for FINALE Is Not A LabVIEW Editor.
+FINALE is a lightweight WebApp to view G Code. FINALE stands for FINALE Is Not A LabVIEW Editor.
 
 This solves many use cases like:
-  - Code sharing : Sharing code with a person with no LV license necessitates printing/taking screenshot.
+  - Code sharing: Sharing code with a person with no LV license necessitates printing/taking screenshot.
```

Comment:
Avoid the "LV" terminology, use LabVIEW instead.

Reply:
Note that we now have a "LabVIEW Community" edition which does not require buying a license. Hence a better justification might be to view something quickly (on a machine which does not have LabVIEW installed, or without wanting to launch LabVIEW).

Reply:
Rephrased.